modules/wildfly/src/main/modules/ogm/infinispan/module.xml

```diff
@@ -10,11 +10,10 @@
         <resource-root path="hibernate-ogm-infinispan-${project.version}.jar" />
     </resources>
     <dependencies>
-        <module name="org.hibernate" export="true" services="import" slot="${hibernate.ogm.module.slot}" />
+        <module name="org.hibernate" slot="${hibernate.ogm.module.slot}" />
         <module name="org.hibernate.hql" slot="${hibernate.hql.module.slot}" />
 
-        <module name="org.infinispan" optional="true" slot="${infinispan.module.slot}" />
-        <module name="org.infinispan.commons" optional="true" slot="${infinispan.module.slot}" />
+        <module name="org.infinispan" slot="${infinispan.module.slot}" />
```

Comment:
You just nailed the very reason for which I started to review them all ;-)
BTW my initial patch was quite more aggressive, but was getting complex quickly. Some more polishing will be possible with some more in-depth refactoring with how extensions and classloaders are handled, for example I don't think any of our modules should export resources as the end user couldn't care less of those.